core-bundle/src/Command/Backup/BackupRestoreCommand.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Command\Backup;
+
+use Contao\CoreBundle\Doctrine\Backup\BackupManagerException;
+use Contao\CoreBundle\Doctrine\Backup\Config\RestoreConfig;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @internal
+ */
+class BackupRestoreCommand extends AbstractBackupCommand
+{
+    protected static $defaultName = 'contao:backup:restore';
+
+    protected function configure(): void
+    {
+        parent::configure();
+
+        $this
+            ->addOption('force', null, InputOption::VALUE_NONE, 'By default, this command checks whether the backup has been generated with Contao too. Use --force to disable this check.')
```

Comment:
Maybe we should explicitly name the command, so that people do not confuse it with extensions?
```suggestion
            ->addOption('force', null, InputOption::VALUE_NONE, 'By default, this command makes sure the backup has been generated with "contao:backup:create". Use --force to disable this check.')
```